<?xml version="1.0" encoding="utf-8"?>

<launch>
  <!-- Console launch prefix -->
  <arg name="launch_prefix" default=""/>

  <!-- Config and weights folder. -->
  <arg name="curr_folder" default="$(find darknet_ros)"/>
  <arg name="weight_file" default="$(find darknet_ros)/params_config/yolo_weights/yolov3.weights"/>
  <arg name="config_file" default="$(find darknet_ros)/params_config/yolo_configs/yolov3.cfg"/>

  <!-- Load parameters -->
  <rosparam command="load" ns="darknet_ros" file="$(find darknet_ros)/params_config/yolo_ros/yolo_ros.yaml"/>

  <!-- Start darknet and ros wrapper -->
  <node pkg="darknet_ros" type="darknet_ros" name="darknet_ros" output="screen" launch-prefix="$(arg launch_prefix)">
    <param name="curr_folder" value="$(arg curr_folder)" />
    <param name="config_file" value="$(arg config_file)" />
    <param name="weight_file" value="$(arg weight_file)" />
  </node>

 <!--<node name="republish" type="republish" pkg="image_transport" output="screen" 	args="compressed in:=/front_camera/image_raw raw out:=/camera/image_raw" /> -->
</launch>
